analyzers/tests/SonarAnalyzer.UnitTest/TestCases/WebConfig/Foo/FilesToAnalyze.txt

```diff
@@ -1 +0,0 @@
-﻿TestCases\WebConfig\Foo\Web.config
```

Comment:
Moving the files into a rule specific directory made a mess in the commit history. Can you re-do this to get proper git `Rename` flag on those? In one step, do the move. In second step, fix the content inside.

Or maybe even better, remove `FilesToAnalyze.txt` and write them to disk in the test scaffolding phase when there's only one file that exists (at least on them points to a file that doesn't exist.